_studio/mfx_lib/decode/mjpeg/src/mfx_mjpeg_dec_decode.cpp

```diff
@@ -1095,8 +1085,7 @@ bool MFX_JPEG_Utility::IsNeedPartialAcceleration(VideoCORE * core, mfxVideoParam
 
         if (par->mfx.JPEGColorFormat == MFX_JPEG_COLORFORMAT_YCbCr &&
             par->mfx.JPEGChromaFormat == MFX_CHROMAFORMAT_YUV420 &&
-            par->mfx.FrameInfo.FourCC == MFX_FOURCC_RGB4 &&
-            (par->mfx.FrameInfo.Width > 4096 || par->mfx.FrameInfo.Height > 4096))
+            par->mfx.FrameInfo.FourCC == MFX_FOURCC_RGB4)
```

Comment:
Does driver really report about such configuration?

Reply:
@otsirule , driver should report 16K resolution on SKL and upper platforms (check on CFL, SKL). This is normal, because resolution support is 16K in readme to media-driver.

Reply:
But this condition not only about resolution but also about FourCC. Why resolution support should influence complex conditions like that?

Reply:
@otsirule , I corrected